Exonoesis/Helping-Hand
# Level transitions

## What is the purpose of these changes?
Games traditionally have a world split into different sections called levels. Our game is also structured in this manner, and as such, we needed a way to transition out of one level and into another.

## What has changed?
We now have tiles that when interacted via the "E" key will transition from the first testing level (outside) to the second testing level (inside). We also fixed the movement stuttering.

## How can I verify this works?
1. Run the code and press play to enter the overworld of Level_0.
2. Use WASD to navigate to the front of the cabin and position the player directly south of the door.
3. Press the "E" key.
4. See that the level has changed to that of Level_2, the inside of the cabin.
5. Hear that the music has changed.
6. Feel free to explore this location and see that all the collision works.
7. Enter the doorway and press E again to return to Level_0.
---
Closes #37
